src/main/java/seedu/module/ui/TaskListPanel.java

```diff
@@ -62,7 +62,7 @@ protected void updateItem(Task task, boolean empty) {
                 } else if (colorToAssign.equals("Yellow")) {
                     addedTaskCard.getRoot().setStyle("-fx-background-color: #F0E68C; ");
                 } else {
-                    addedTaskCard.getRoot().setStyle("-fx-background-color: #DC143C; ");
+                    addedTaskCard.getRoot().setStyle("-fx-background-color: #FF6347; ");
```

Comment:
I think the UG `Features` section should also be updated to describe the task colours.